packages/orchestration/src/orchestration-api.ts

```diff
@@ -172,6 +170,20 @@ export interface OrchestrationAccountI {
    * @returns void
    */
   transferSteps: (amount: AmountArg, msg: TransferMsg) => Promise<void>;
+
+  /**
+   * Returns `invitationMakers` and `publicSubscribers` to the account
+   * holder's smart wallet so they can continue interacting with the account
+   * and read account state in vstorage if published.
+   */
+  asContinuingOffer: () => Promise<ContinuingOfferResult>;
+  /**
+   * Public topics are a map to different vstorage paths and subscribers that
+   * can be shared with on or offchain clients.
+   * When returned as part of a continuing invitation, it will appear
+   * in the {@link CurrentWalletRecord} in vstorage.
+   */
+  getPublicTopics: () => Promise<TopicsRecord>;
```

Comment:
these are top level API changes that should have @dtribble approval. I approve and will leave to you how/when to sync with Dean